apps/nextjs-app/src/features/app/blocks/view/list/useDuplicateView.ts

```diff
@@ -0,0 +1,37 @@
+import type { IViewInstance } from '@teable/sdk';
+import { useTable } from '@teable/sdk/hooks';
+import { pick } from 'lodash';
+import { useRouter } from 'next/router';
+import { useCallback } from 'react';
+
+export function useDuplicateView(view: IViewInstance) {
+  const table = useTable();
+  const router = useRouter();
+
+  const newView = pick(view, [
+    'type',
+    'columnMeta',
+    'description',
+    'filter',
+    'group',
+    'name',
```

Comment:
should append an auto-increase index to the name.
name -> name 1
name 1 -> name 2

Reply:
it's in the backend

Reply:
@tea-artist

[video-2024-12-06_08.30.36.webm](https://github.com/user-attachments/assets/4c680a81-9522-494e-b792-78e9cf37f9ad)